src/path_manager.c

```diff
@@ -1032,18 +1096,36 @@ struct mptcpd_pm *mptcpd_pm_create(struct mptcpd_config const *config)
           Warn the user if the MPTCP generic netlink family doesn't
           appear within a reasonable amount of time.
         */
-        pm->timeout =
+        pm->family_timeout =
                 l_timeout_create(FAMILY_TIMEOUT_SECONDS,
                                  family_timeout,
                                  pm,
                                  NULL);
 
-        if (pm->timeout == NULL) {
+        if (pm->family_timeout == NULL) {
+                mptcpd_pm_destroy(pm);
+                l_error("Unable to create timeout handler.");
+                return NULL;
+        }
+
+        /*
+          Periodically synchronize kernel state with mptcpd, such as
+          address IDs managed by the in-kernel path manager.
+        */
+        pm->sync_timeout =
```

Comment:
What do you think about using `sync_interval == 0` to turn off polling?

That seems to be what happens with `l_timeout_create()` is given a 0 time value anyway, but it goes ahead and sets up the `timerfd` and ELL watch anyway. Could skip that work if the sync interval is 0.

Reply:
I agree.  Implemented in 69b1e6e.